doc-Managing_Infrastructure_and_Inventory/topics/Deleting_EBS_Volumes.adoc

```diff
@@ -1,11 +1,12 @@
-[[deleting_cinder_volumes]]
+[[deleting_ebs_volumes]]
 = Deleting a Volume
```

Comment:
@dayleparker - There is no difference in this procedure in terms of functionality right now, but we cannot reference the same section twice in the same guide unless we change the xref for both references. The OSP side is tied up in the quickstart for OSP guide, so I'm a little hesitant to make changes on that level at this stage. What do you think?